scheduler/driver.go

```diff
@@ -19,9 +24,45 @@ func getFrameworkID() *mesos.FrameworkID {
 	return nil
 }
 
+func getPrincipalID(credential *mesos.Credential) *string {
+	if credential != nil {
+		return credential.Principal
+	}
+	return nil
+}
+
+func getCredential() (*mesos.Credential, error) {
+	if viper.IsSet("credential_file") {
+		content, err := ioutil.ReadFile(viper.GetString("credential_file"))
+		if err != nil {
+			logrus.WithError(err).WithFields(logrus.Fields{
+				"credential_file": viper.GetString("credential_file"),
+			}).Error("Unable to read credential_file")
+			return nil, err
+		}
+		fields := strings.Fields(string(content))
+		logrus.WithField("principal", fields[0]).Info("Successfully loaded principal")
+		return &mesos.Credential{
+			Principal: proto.String(fields[0]),
+			Secret:    proto.String(fields[1]),
+		}, nil
+	}
+	logrus.Info("No credentials specified in configuration")
```

Comment:
I'd probably prefer to have this as a debug message. Granted, it probably won't be very spammy, but I'd say it's not really relevant in a non-debug scenario.